### PR TITLE
Adding `spin-registry` support

### PR DESCRIPTION
**Pull request by macolso**
Content must go through a pre-merge checklist.

## Pre-Merge Content Checklist

This documentation has been checked to ensure that:

- [x] The `title, `template`, and `date` are all set
- [x] File does not use CRLF, but uses plain LF (hint: use `cat -ve <filename> | grep '^M' | wc -l` and expect 0 as a result) 
- [x] Has passed [`bart check`](https://developer.fermyon.com/bartholomew/quickstart)

Screenshot 2023-02-17 at 1 27 38 pm

- [x] Has run `npm run build-index`
- [x] Has been manually tested by running in Spin/Bartholomew (hint: use `PREVIEW_MODE=1` and run `npm run styles` to update styling)

Screenshot 2023-02-17 at 1 38 02 pm

- [x] Headings are using Title Case
- [x] Code blocks have the programming language set to properly highlight syntax and the proper copy directive
- [x] Have tested with `npm run test` and resolved all errors

Screenshot 2023-02-17 at 1 40 08 pm

### CODE REVIEW

**Comment by macolso** [comment]
asking @fibonacci1729 for a fact-checking review on the 0.9 syntax

File: content/spin/spin-oci.md [hunk 1]
```diff
@@ -58,33 +61,91 @@ Now, switch to that directory and rebuild the application.
...
+
+{{ tabs "spin-version" }}
+
+{{ startTab "v0.8.0"}}
+
  <!-- @selectiveCpy -->
 
  ```bash
  # make sure you've built your application with `spin build` prior
  spin oci run ghcr.io/USERNAME/spin-react-fullstack
```
Comment (radu-matei) [inline content/spin/spin-oci.md:131]: This also needs the tag.

File: content/spin/spin-oci.md [hunk 2]
```diff
@@ -58,33 +61,91 @@ Now, switch to that directory and rebuild the application.
...
 ```
 
+{{ blockEnd }}
+
+{{ startTab "v0.9.0"}}
+
+ <!-- @selectiveCpy -->
+ 
+ ```bash
+ spin registry pull ghcr.io/USERNAME/spin-react-fullstack
```
Comment (radu-matei) [inline content/spin/spin-oci.md:112]: This also needs the tag.

File: content/spin/spin-oci.md [hunk 3]
```diff
@@ -58,33 +61,91 @@ Now, switch to that directory and rebuild the application.
...
 
+{{ blockEnd }}
+
+{{ startTab "v0.9.0"}}
+
+<!-- @selectiveCpy -->
+
+ ```bash
+ # make sure you've built your application with `spin build` prior
+ spin up --from-registry ghcr.io/USERNAME/spin-react-fullstack
```
Comment (radu-matei) [inline content/spin/spin-oci.md:142]: This also needs the tag.

File: content/spin/spin-oci.md [hunk 4]
```diff
@@ -28,6 +29,8 @@ First, follow [this guide](./install.md) to install Spin v0.8.0 or later. To ens
 $ spin --version
 ```
 
+Please note if you're using version v0.9.0 or later you will be using the command `spin registry`. The command `spin oci` will only be supported with v0.8.8.
```
[the remark this answers is not in the document]
Comment (tpmccallum) [reply content/spin/spin-oci.md:32]: Looks like https://github.com/fermyon/developer/commit/6e163fbabea14119032c7a7455bd1882bda2393e already covered this off.

Comment (tpmccallum) [inline content/spin/spin-oci.md:32]:
```suggestion
If you're using version v0.9.0 or later, it is officially recommended that you use the command `spin registry`. However, please note that the command `spin oci` (whilst initially supported by v0.8.0) is now also an alias (`oci` and `registry` are interchangeable) in v0.9.0. 
```

File: content/spin/spin-oci.md (hunk 1 above)
Comment (tpmccallum) [inline content/spin/spin-oci.md:131]:
```suggestion
 spin up --from-registry ghcr.io/USERNAME/spin-react-fullstack:v1
```

File: content/spin/spin-oci.md (hunk 2 above)
Comment (tpmccallum) [inline content/spin/spin-oci.md:112]:
```suggestion
 spin registry pull ghcr.io/USERNAME/spin-react-fullstack:v1
```

File: content/spin/spin-oci.md (hunk 3 above)
Comment (tpmccallum) [inline content/spin/spin-oci.md:142]:
```suggestion
 spin up --from-registry ghcr.io/USERNAME/spin-react-fullstack:v1
```

File: content/spin/spin-oci.md (hunk 2 above)
Comment (tpmccallum) [reply content/spin/spin-oci.md:112]: Thanks @radu-matei

File: content/spin/spin-oci.md (hunk 1 above)
Comment (tpmccallum) [reply content/spin/spin-oci.md:131]: Thanks @radu-matei

File: content/spin/spin-oci.md (hunk 3 above)
Comment (tpmccallum) [reply content/spin/spin-oci.md:142]: Thanks @radu-matei

File: content/spin/spin-oci.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (tpmccallum) [reply content/spin/spin-oci.md:32]: Thanks @radu-matei ; all fixed.

**Review by tpmccallum — APPROVED** [review]
Looks great @macolso

**Comment by tpmccallum** [comment]
@macolso - A thousand apologies; I somehow messed up this PR by loosing myself in Git commands about generating package-lock files and so forth. I decided it would be best to start again on a new branch and therefore please find duplicate of your great work over on https://github.com/fermyon/developer/pull/349

I will close this for now and would appreciate any feedback and/or an approval over on 349

Thank you so much for this work; love the multi code blocks btw.

**Pull request closed without merging**